tests/core_tests.py

```diff
@@ -373,13 +373,30 @@ def test_queries_endpoint(self):
         self.assertEquals(403, resp.status_code)
 
     def test_search_query_endpoint(self):
-        userId = 'userId=null'
-        databaseId = 'databaseId=null'
-        searchText = 'searchText=null'
+        self.run_sql("SELECT * FROM ab_user", 'admin', client_id='client_id_1')
+        self.login('admin')
+        first_query = db.session.query(models.Query).filter_by(
+            sql='SELECT * FROM ab_user').first()
+        database_id = 'database_id=1'
         status = 'status=success'
-        params = [userId, databaseId, searchText, status]
-        resp = self.client.get('/caravel/search_queries?'+'&'.join(params))
-        self.assertEquals(200, resp.status_code)
+        params = [database_id, status]
+        resp = self.get_resp('/caravel/search_queries?'+'&'.join(params))
+        data = json.loads(resp)
+        self.assertEquals(1, len(data))
+
+        self.run_sql("SELECT * FROM ab_permission", 'admin', client_id='client_id_2')
+        second_query = db.session.query(models.Query).filter_by(
+            sql='SELECT * FROM ab_permission').first()
+
+        from_time = 'from=' + \
+            str(int(first_query.start_time))
+        to_time = 'to=' + \
+            str(int(second_query.start_time))
+        params = [from_time, to_time]
+        resp = self.get_resp('/caravel/search_queries?'+'&'.join(params))
+        data = json.loads(resp)
+        self.assertEquals(1, len(data))
+        self.logout()
```

Comment:
please test multiple use cases:
make sure that initially you have queries for 2 users, 1 user has queries for at least 2 databases

Test that:
- time filter filters out queries both for `to` and `from` use cases
- user_id filters given user queries
- QUERY_SEARCH_LIMIT works
- like query returns the expected queries

It would be nice to have all them as separate test case and you can you SetUp method to create initial dataset. Come by if you need some help with it.